client/core/esx.lua

```diff
@@ -21,39 +20,48 @@ end)
 
 local shared = exports[coreName]:getSharedObject()
 
-local coreFunctionsOverride = {
-    playerData = {
-        originalMethod = 'GetPlayerData',
-        modifier = {
-            executeFun = true,
-            effect = function(originalFun)
-                while not shared.IsPlayerLoaded() do
-                    Wait(1000)
-                end
-                local data = originalFun()
-                local job = data.job
-
-                local month, day, year = data.dateofbirth and data.dateofbirth:match("(%d+)/(%d+)/(%d+)") or 00, 00, 24
-
-                return {
-                    cid = data.identifier,
-                    money = data.money or 0,
-                    inventory = data.inventory or {},
-                    job = {name = job.name, label = job.label, onDuty = true, isBoss = false, grade = {name = job.grade, label = job.grade_label, salary = job.grade_salary}},
-                    firstName = data.firstName or 'Unknown',
-                    lastName = data.lastName or 'Unknown',
-                    phone = data.phone_number or '0',
-                    gender = data.sex == 'm' and 'male' or 'female',
-                    dob = ('%s/%s/%s'):format(month, day, year)
-                }
-            end
+function Core.getPlayerData()
+    local player = shared.GetPlayerData()
+    local job = player.job
+    local gang = player.gang
+    local charinfo = player.charinfo
+    local year, month, day = charinfo.birthdate:match('(%d+)-(%d+)-(%d+)')
+
+    local formattedJob = {
+        name = job.name,
+        label = job.label,
+        onDuty = job.onduty,
+        isBoss = job.isboss,
```

Comment:
The table structure is not matching what was previously returned by the coreFunctionsOverride. Please check

Reply:
ahhmy bad i thought its same doing changes